scripts/create-rpms.sh

```diff
@@ -7,6 +7,13 @@ if [ $? -eq 1 ]; then
 	exit 1
 fi
 
+#check if rpmdevtools is installed
+yum list installed rpmdevtools
+if [ $? -eq 1 ]; then
```

Comment:
The success checks would be a bit safer if coded like..
```bash
if [ $? -ne 0 ]; then
```

Reply:
good point.